source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/__tests__/acceptance-test.tsx

```diff
@@ -94,12 +99,11 @@ describe('Acceptance | Archive Project', () => {
             attributes: {
               'date-archived': null
             },
-            relationships: {},
           }});
       });
 
       describe('the reactivate button is clicked', () => {
-        beforeEach(async () => {
+        beforeEach(async function() {
```

Comment:
this is not part of the scope of this fix, was it a blocker?

Reply:
I believe an intermittent failure issue